components/registry/org.wso2.carbon.registry.extensions/src/main/java/org/wso2/carbon/registry/extensions/handlers/utils/SwaggerProcessor.java

```diff
@@ -72,14 +73,30 @@
 	private String swaggerResourcesPath;
 	private String documentVersion;
 	private String endpointUrl;
-	OMElement restServiceElement = null;
-	OMElement endpointElement = null;
+	private OMElement restServiceElement = null;
+	private OMElement endpointElement = null;
 	private String endpointLocation;
+	private boolean createRestServiceArtifact;
 
-	public SwaggerProcessor(RequestContext requestContext) {
+	public SwaggerProcessor(RequestContext requestContext, boolean createRestServiceArtifact) {
 		this.parser = new JsonParser();
 		this.requestContext = requestContext;
 		this.registry = requestContext.getRegistry();
+		this.createRestServiceArtifact = createRestServiceArtifact;
+	}
+
+	/**
+	 * @return createRestServiceArtifact
+	 */
+	public boolean isCreateRestServiceArtifact() {
+		return createRestServiceArtifact;
+	}
```

Comment:
@daneshk This issue was due to usage of tab character instead of 4 spaces. converted the file to use 4 spaces and reformatted the whole file in #218 (6917e4f)